code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -172,6 +172,11 @@
 	description = "Water blessed by some deity."
 	color = "#E0E8EF" // rgb: 224, 232, 239
 
+/datum/reagent/water/holywater/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(is_servant_of_ratvar(M))
+		M << "<span class='userdanger'>A darkness begins to spread its unholy tendrils through your mind, purging the Justiciar's influence!</span>"
```

Comment:
Nah, remember, this is from the perspective of a follower still. They haven't been deconverted yet. If you look at cult, their minds have been tainted with narsie's dark power. Ratvarian followers convert because their minds are filled with knowledge of ratvar. Deconversion removes that knowledge.
It's why the emote fluff messages show the followers are absolutely terrified of the deconversion. It is unholy to them and moves them closer to narsie.
Basically, think of ratvar and narsie as polar opposites of a spectrum, with neutral crewmembers in the middle.